apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx

```diff
@@ -255,7 +254,7 @@ export const AttributeSelection: FunctionComponent<AttributeSelectionPropsInterf
     /**
      * Check whether claim is mandatory or not
      *
-     * @param uri Claim URI to be checked.
+     * @param uri - Claim URI to be checked.
```

Comment:
Can we add return type as well to the comment.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3440/commits/d16981abb35182947ec076270163483cb6bd55f6